samcli/local/apigw/local_apigw_service.py

```diff
@@ -287,25 +287,21 @@ def _parse_lambda_output(lambda_output, binary_types, flask_request):
             if status_code <= 0:
                 raise ValueError
         except ValueError:
-            message = "statusCode must be a positive int"
-            LOG.error(message)
-            raise LambdaResponseParseException(message)
+            raise LambdaResponseParseException("statusCode must be a positive int")
 
         try:
             if body:
                 body = str(body)
         except ValueError:
-            message = "Non null response bodies should be able to convert to string: {}".format(body)
-            LOG.error(message)
-            raise LambdaResponseParseException(message)
+            raise LambdaResponseParseException(f"Non null response bodies should be able to convert to string: {body}")
 
         # API Gateway only accepts statusCode, body, headers, and isBase64Encoded in
         # a response shape.
         invalid_keys = LocalApigwService._invalid_apig_response_keys(json_output)
         if bool(invalid_keys):
-            msg = "Invalid API Gateway Response Keys: " + str(invalid_keys) + " in " + str(json_output)
-            LOG.error(msg)
-            raise LambdaResponseParseException(msg)
+            raise LambdaResponseParseException(
+                f"Invalid API Gateway Response Keys: {str(invalid_keys)} in {str(json_output)}"
```

Comment:
No need call `str()`. F-strings will do this for you.

Reply:
Oh didn't know that, thanks! :) Will update in next revision